source/shared/cs_MicroappStructs.h

```diff
@@ -245,6 +262,15 @@ struct __attribute__((packed)) microapp_pin_cmd_t {
 static_assert(sizeof(microapp_pin_cmd_t) == MICROAPP_PIN_CMD_SIZE_HEADER);
 static_assert(sizeof(microapp_pin_cmd_t) <= MAX_PAYLOAD);
 
+/*
+ * Struct for controlling the dimmer and switch
+ */
+struct __attribute__((packed)) microapp_dimmer_switch_cmd_t {
+	microapp_cmd_t header;
+	uint8_t opcode; // CommandMicroappDimmerSwitchOpcode
```

Comment:
Preference for comments on lines separate from the code. If you want to explain the members of a struct, feel free to do so in the comment section preceding the struct.